apps/user-office-frontend-e2e/cypress/e2e/peopleTable.cy.ts

```diff
@@ -526,4 +526,32 @@ context('PageTable component tests', () => {
       cy.contains('1 user(s) selected').should('exist');
     });
   });
+
+  describe('PeopleTable component allows swapping between PIs and CoIs', () => {
+    it('Should allow user to swap between PIs and CoIs', () => {
+      cy.login('user1');
+      cy.visit('/');
+
+      cy.contains('New Proposal').click();
+      cy.get('[data-cy=call-list]').find('li:first-child').click();
+
+      cy.get('[data-cy=add-participant-button]').click();
+
+      cy.get('[role="presentation"] [role="dialog"]').as('modal');
```

Comment:
This line might not be needed because I don't see `modal` usage anywhere bellow in this test case.